AUTHORS.txt

```diff
@@ -1,4 +1,4 @@
-* `Agustin Henze <https://github.com/agustinhenze>`_
+﻿* `Agustin Henze <https://github.com/agustinhenze>`_
```

Comment:
You inserted a UTF-8 BOM here (a hexdump shows that the file now starts with `ef bb bf`).

Reply:
(https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8)

Reply:
I fixed this for now. In the future, please make sure your editor does not insert BOMs (never use notepad.exe to edit stuff like that)

https://en.wikipedia.org/wiki/Byte_order_mark